code/game/objects/items/weapons/thurible.dm

```diff
@@ -164,11 +164,11 @@
 			var/mob/living/carbon/C = A
 			if(C.can_breathe_gas())
 				mobs_to_smoke += C
+	if(mobs_to_smoke)
```

Comment:
This will always be truthy otherwise
```suggestion
	if(length(mobs_to_smoke))
```

Reply:
An empty list wont default to false? It would runtime during my testing if it were always true.

Reply:
![image](https://github.com/user-attachments/assets/359812e6-042c-43da-a2f5-91072bd79ff0)
BYOND isnt sane...